include/deal.II/grid/grid_tools.h

```diff
@@ -263,6 +263,19 @@ namespace GridTools
     const Iterator &                                      object,
     const Point<Iterator::AccessorType::space_dimension> &trial_point);
 
+  /**
+   * Return the arrays that define the coarse mesh of a Triangulation. This
+   * function is the inverse of Triangulation::create_triangulation.
```

Comment:
In the majority of cases, we write `Class::function()` (i.e., using parentheses) to indicate functions.

Same in the changelog entry.

Reply:
Fixed